tasks/CentOS/redis.yml

```diff
@@ -4,6 +4,12 @@
 
   - include_vars: "{{ ansible_distribution }}.yml"
 
+  - name: Install EPEL repo
+    yum:
+      name: epel-release
+      state: present
+    when: enable_epel_repo == true
```

Comment:
Hey @romainrbr - You can drop the `== true` for the when statements, Ansible should automatically eval the variable and if it's a boolean, this should just work.